tests/test_requirements.txt

```diff
@@ -0,0 +1,3 @@
+nox==2025.2.9
+pytest==8.3.4
+pyenv==2.4.10
```

Comment:
This won't work :(

```
$ pip install -r tests/test_requirements.txt 
Collecting nox==2025.2.9 (from -r tests/test_requirements.txt (line 1))
  Downloading nox-2025.2.9-py3-none-any.whl.metadata (4.7 kB)
Collecting pytest==8.3.4 (from -r tests/test_requirements.txt (line 2))
  Using cached pytest-8.3.4-py3-none-any.whl.metadata (7.5 kB)
ERROR: Could not find a version that satisfies the requirement pyenv==2.4.10 (from versions: 0.0.0, 0.0.1)
```

https://pypi.org/project/pyenv/

`NOTE: This is a placeholder package. pyenv is a collection of shell scripts and not installable with pip.`

Reply:
Thanks, Ivica! I will be working on it.

Reply:
Hi @ivica-k,  
I wrote a Makefile that includes:  
- Verifying that `pyenv` is installed and installing it if it's not  
- Installing the required dependencies  
- Installing the specified Python version  
- Configuring the global Python version  
- Running `nox`